apis/dkjson

```diff
@@ -11,7 +11,7 @@ David Kolf's JSON module for Lua 5.1/5.2
 This module writes no global values, not even the module table.
 Import it using
 
-    json = require ("dkjson")
+    json = require ("github.rom/apis/dkjson")
```

Comment:
You realize that the line you changed here doesn't even execute, and`require` doesn't exist in CC, right?